modules/fts/pages/fts-searching-with-the-rest-api.adoc

```diff
@@ -100,6 +100,37 @@ http://localhost:18094/api/index/travel-sample-index/query \
 For additional information on secure communication with Couchbase Server, see xref:manage:manage-security/manage-connections-and-disks.adoc[Manage Connections and Disks].
 
 
+[#searching-with-query-timeouts]
+== Searching with timeouts
+
+Once the the index becomes really large, it is possible that certain search requests may take
+longer time to finish the execution.Such slow responses would affect the query latencies as well as the resource consumption on the server side.
+
+For avoiding such unplesant situation, users can pass a search "timeout" value in the request.
+By specificing timeout parameters in the seach request, users are bound to get a response within the stipulated time.
+Here the timeout value is specified in unit of milliseconds.
+
+If the search request times out before gathering all results, then the partial results will be
+returned to the user with detailed partition level search result status.
```

Comment:
What could be useful here is an output snippet indicating the partial search status too? We can achieve this by placing sleeps in the code.

Reply:
Thought of that, but then realised that once the error comes, it's natural for the user to interpret it like any other response.  Response always follows the same pattern and describing that in its respective section ought to be enough is my thinking.
The intent here was more on - there is an option to do this and this is how you do it..